### ISSUE TITLE
How to develop a portfolio to help me find remote work

### ISSUE THREAD

**Issue by MaraAlexa** [IonicaBizau/ama#6]
I would like to write code and work remotely. What kind of steps should I take to make it happen? 
I think a good online portfolio would increase my chances to find some remote work. What do you think should be in the portfolio? What makes for a good candidate? How did you achieve all of this?
Any other practical advice would be also greatly appreciated.

Thanks

**Comment by sureshatta** [comment]
First thing I suggest is to build a fat profile on stackoverflow. That fetches more strength and respect on other paid work portals.

Build some reputation before you start asking work online. Period.

**Comment by IonicaBizau** [comment]
**TL;DR**: Build. Publish. Do open-source. Help others. Profit. Following Jesus matters.

---

Think at nice apps that could help the world around you. Start with little things. Write a lot of code. Modularize it. Publish the modules on GitHub (one module, one repository). While doing all this stuff, you will obviously learn a lot.

When you're stuck, ask what you don't know on StackOverflow ([this is how I do it](https://github.com/IonicaBizau/ama/issues/4#issuecomment-191780199)). Try to answer questions posted by other people. While asking and answering you will learn a lot and as bonus, you will build a great SO profile like @sureshatta mentioned.

Start making some business around this: I started with a free website for someone, then I increased the price while gaining experience because I had something to show people what I built in the past. :smile: 

This is how I started: I found a nice site with a terrible design and I emailed that guy saying I want to help him with a better site for free. We did it. He sent me a little gift and I was overjoyed. After few years, this website is still working and I can show it to you: [`balonro.ro`](http://balonro.ro/)

[![image](https://cloud.githubusercontent.com/assets/2864371/13548890/3f482512-e303-11e5-8eec-db1f18716243.png)](http://balonro.ro/)

Of course, it takes time, but try to build things and profit while doing that. I got hired by @jillix in 2012—at the time I had no idea what Node.js is and I knew only a bit of JavaScript. But at least, I had something to show (some Wordpress sites, C# apps etc). They didn't regret they hired me. :stuck_out_tongue_winking_eye:

> What do you think should be in the portfolio?

It depends what you want to work on. It can be JavaScript libraries, jQuery plugins, CSS modules etc. In fact, anything that helps people around you.

> How did you achieve all of this?

I really like what I do. Of course, it was a lot of work but I ~~was~~ am not alone in this journey.
It would be incorrect to avoid saying that I do believe in God. A God that is REAL. He says:

> *But seek first his kingdom and his righteousness, and all these things will be given to you as well.*
> —Matthew 6:33 

Following Jesus is probably the only thing I'm really proud of. All the thanks go to him and the happiness is ours. :blush:

**Comment by israteneda** [comment]
Amazing answer @IonicaBizau when you started to believe in God and how did occur.
